.gitignore

```diff
@@ -1,9 +1,7 @@
 junit-results.xml
 *.pyc
 *.pyo
-*.sh
 *.so
-*.fif
```

Comment:
At some point a bunch of these files were created by tests, so having them in `.gitignore` was a feature. Is there another option?

We could try to clean up all these instances. If you do `make test-doc` and `pytest mne -m "not ultraslowtest"` locally then do `git status` it might tell you if the presence of these files is still an issue for most use cases. Then the really annoying one would be to do a full doc build locally and see what lingers.

Reply:
Yes there are alternative approaches. I have some ideas.